pkg/kubelet/rkt/rkt_linux.go

```diff
@@ -742,7 +748,7 @@ func (r *runtime) writeDockerAuthConfig(image string, creds docker.AuthConfigura
 //
 // https://github.com/GoogleCloudPlatform/kubernetes/issues/7203
 //
-func (r *runtime) PullImage(image kubecontainer.ImageSpec, _ []api.Secret) error {
+func (r *runtime) PullImage(image kubecontainer.ImageSpec, pullSecrets []api.Secret) error {
```

Comment:
> Are you planning on making this func use pullSecrets in this PR?

rebase movement.  Refactored to use it more properly.  It's still only first hit wins (as it was before).